src/test/configs/emptyConfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "monitoring": {
+        "metrics": {
+            "enable": true,
+            "metrics_list": [
+                "ovms_current_requests",
+                "ovms_infer_req_active",
```

Comment:
The config name "empty config" does not reflect its content - metrics are enabled

Reply:
Idea was that it was just empty but I will rename it to noServablesMetricsEnabled